package.json

```diff
@@ -71,9 +71,10 @@
     "fs-extra": "^7.0.1",
     "lerna-changelog": "^0.8.2",
     "loader.js": "^4.7.0",
-    "mocha": "^6.0.0",
-    "prettier": "^1.16.4",
-    "walk-sync": "^1.1.3"
+    "mocha": "^5.2.0",
```

Comment:
What is the rationale for downgrading all three of these addons to earlier versions?

Reply:
Fixed this, I undo these changes